plugin/core/views.py

```diff
@@ -327,6 +327,10 @@ def position(view: sublime.View, offset: int) -> Position:
     return offset_to_point(view, offset).to_lsp()
 
 
+def word(view: sublime.View, offset: int) -> str:
+    return view.substr(view.expand_by_class(offset, sublime.CLASS_WORD_START | sublime.CLASS_WORD_END, "[]{}()<>:."))
```

Comment:
This is the same what Default/symbol.py uses at a first attempt to get the word at the current cursor position.

I'm not sure why exactly this is preferred over `view.word()`, but I thought it would be better to keep it consistent with the builtin implementation. It would for example include string quotation marks, while `view.word()` would strap them off.

Reply:
Is it actually consistent with builtin implementation? Check the placeholder text in the second case, when the cursor is a the beginning of the class property:

https://user-images.githubusercontent.com/153197/199826271-2e387b01-a6f3-4158-a4b6-15ecc868d1df.mov

Built-in implementation can't actually show references for this symbol but it correctly shows "Unable to find capability" in the status field.

Reply:
I guess looking at default implementation explains it:

```py
def reference_at_point(view, pt):
    symbol = view.substr(view.expand_by_class(pt, sublime.CLASS_WORD_START | sublime.CLASS_WORD_END, "[]{}()<>:."))
    locations = lookup_references(view.window(), symbol)

    if len(locations) == 0:
        symbol = view.substr(view.word(pt))
        locations = lookup_references(view.window(), symbol)

    return symbol, locations
```